Sources/flintc/main.swift

```diff
@@ -14,8 +14,9 @@ func main() {
     Flag("dump-ast", flag: "a", description: "Print the abstract syntax tree of the code."),
     Flag("verify", flag: "v", description: "Verify expected diagnostics were produced."),
     Flag("quiet", flag: "q", description: "Supress warnings and only emit fatal errors."),
+    Flag("no-stdlib", flag: "S", description: "Do not load the standard library"),
```

Comment:
I would get rid of the short flags unless they are 100% obvious (e.g. `-v` is `--version`).

Reply:
#64